docs/design/features/hybrid-globalization.md

```diff
@@ -198,3 +198,115 @@ Web API does not expose locale-sensitive endsWith/startsWith function. As a work
+
+- `IgnoreKanaType` is not supported because there is no equivalent in native api. Throws `PlatformNotSupportedException`.
+
+- `None`:
+
+`CompareOptions.None` is mapped to `NSStringCompareOptions.NSLiteralSearch`
+
+There are some behaviour changes. Below are examples of such cases.
+
+| **character 1** | **character 2** | **CompareOptions** | **hybrid globalization** | **icu** |                       **comments**                      |
+|:---------------:|:---------------:|--------------------|:------------------------:|:-------:|:-------------------------------------------------------:|
+|   `\u3042` あ   |   `\u30A1` ァ   |   None  |             1            |    -1   |     hiragana and katakana characters are ordered differently compared to ICU    |
+|   `\u304D\u3083` きゃ  |   `\u30AD\u30E3` キャ |     None     |             1            |    -1   | hiragana and katakana characters are ordered differently compared to ICU  |
+|   `\u304D\u3083` きゃ  |   `\u30AD\u3083` キゃ  |     None     |             1           |    -1   |  hiragana and katakana characters are ordered differently compared to ICU  |
+|   `\u3070\u3073\uFF8C\uFF9E\uFF8D\uFF9E\u307C` ばびﾌﾞﾍﾞぼ  |   `\u30D0\u30D3\u3076\u30D9\uFF8E\uFF9E` バビぶベﾎﾞ  |     None     |   1  |  -1  | hiragana and katakana characters are ordered differently compared to ICU   |
+|   `\u3060` だ  |   `\u30C0` ダ  |     None     |   1  |  -1  |   hiragana and katakana characters are ordered differently compared to ICU |
+|   `\u00C0` À  |   `A\u0300` À  |     None     |   1  |  0  |   This is not same character for native api |
+
+- `StringSort` shares the same logic as None.
+
+- `IgnoreCase`:
+
+`CompareOptions.IgnoreCase` is mapped to `NSStringCompareOptions.NSCaseInsensitiveSearch`
+
+There are some behaviour changes. Below are examples of such cases.
+
+| **character 1** | **character 2** | **CompareOptions** | **hybrid globalization** | **icu** |                       **comments**                      |
+|:---------------:|:---------------:|--------------------|:------------------------:|:-------:|:-------------------------------------------------------:|
+|   `\u20A9` ₩  |   `\uFFE6` ￦  |     IgnoreCase     |   0  |  -1  |    |
+|   `\uFF66` ｦ  |   `\u30F2` ヲ  |     IgnoreCase     |   0  |  1  |    |
+|   `\u3060` だ |   `\u30C0` ダ  |     IgnoreCase     |   0  |  1  |    |
+
+- `IgnoreNoneSpace`:
+
+`CompareOptions.IgnoreNoneSpace` is mapped to `NSStringCompareOptions.NSDiacriticInsensitiveSearch`
+
+There are some behaviour changes. Below are examples of such cases.
+
+| **character 1** | **character 2** | **CompareOptions** | **hybrid globalization** | **icu** |                       **comments**                      |
+|:---------------:|:---------------:|--------------------|:------------------------:|:-------:|:-------------------------------------------------------:|
+|   `\uFF66` ｦ  |   `\u30F2` ヲ  |     IgnoreNoneSpace     |   0  |  1  |    |
```

Comment:
Are these two also considered equal with IgnoreWidth? It seems strange that the `NSDiacriticInsensitiveSearch` considers these the same.